app/js/components/orderEntry/OrderHeader.jsx

```diff
@@ -1,13 +1,32 @@
 import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
+import IconButton from '../button/IconButton';
 
-const OrderHeader = ({ status, orderable }) => (
+const OrderHeader = ({
+  status,
+  orderable,
+  order,
+  handleEdit,
+}) => (
   <Fragment>
     <th>{orderable}</th>
     <th>{status}</th>
-    <th>
-      <i className="icon-edit" />
-      <i className="icon-delete" />
+    <th className="order-actions-btn">
+      {order.type === 'drugorder'
```

Comment:
Instead of rendering an empty span, can we render `null` or even better 
```
{order.type === 'drugorder'
&& <IconButton
          iconClass="icon-pencil"
          iconTitle="EDIT"
          onClick={handleEdit}
          dataContext={order}
        />  }